src/command.h

```diff
@@ -183,6 +187,52 @@ class DrawRectCommand : public PipelineCommand {
   float height_ = 0.0;
 };
 
+/// Command to draw a grid of recrangles on screen.
+class DrawGridCommand : public PipelineCommand {
+ public:
+  explicit DrawGridCommand(Pipeline* pipeline, PipelineData data);
+  ~DrawGridCommand() override;
+
+  const PipelineData* GetPipelineData() const { return &data_; }
+
+  void EnableOrtho() { is_ortho_ = true; }
+  bool IsOrtho() const { return is_ortho_; }
+
+  void EnablePatch() { is_patch_ = true; }
```

Comment:
Is this needed? Can it be removed?

Reply:
Copied off draw_rect. This is related to tesselation, and may be useful in draw_rect. Removed here.